src/com/xilinx/rapidwright/util/PerformanceExplorer.java

```diff
@@ -314,12 +320,13 @@ public void explorePerformance(){
 		JobQueue jobs = new JobQueue();
 		
 		if(pblocks == null){
-			pblocks = new ArrayList<>();
-			pblocks.add(null);
+			pblocks = new HashMap<>();
+			pblocks.put(null, null);
 		}
 		
-		for(int pb=0; pb < pblocks.size(); pb++){
-			PBlock pblock = pblocks.get(pb);
+		int pb = 0;
```

Comment:
Doesn't look like this is used?

Reply:
This is used as part of the `uniqueID`, (GitHub was hiding it from you).